src/test/java/org/assertj/core/internal/strings/Strings_assertContainsIgnoringWhitespace_Test.java

```diff
@@ -0,0 +1,101 @@
+package org.assertj.core.internal.strings;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.internal.StandardComparisonStrategy;
+import org.assertj.core.internal.Strings;
+import org.assertj.core.internal.StringsBaseTest;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.assertj.core.error.ShouldContainCharSequence.*;
+import static org.assertj.core.internal.ErrorMessages.charSequenceToLookForIsNull;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.Arrays.array;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.assertj.core.util.Sets.newLinkedHashSet;
+
+/**
+ * Tests for <code>{@link Strings#assertContainsIgnoringWhitespace(AssertionInfo, CharSequence, CharSequence...)} </code>.
+ *
+ * @author Johannes Becker
+ */
+public class Strings_assertContainsIgnoringWhitespace_Test extends StringsBaseTest {
```

Comment:
use `expectAssertionerror` + GIVEN WHEN THEN as indicated in the PR guidelines, it makes the tests more readable.

Reply:
Well, that happens when you just copy&paste&modify exsiting test cases. Sorry for that...